purchases/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.revenuecat.purchases">
+    package="com.revenuecat.purchases.api">
```

Comment:
we can't have two modules with the same package. I am naming this one `api` because it's the entry point to our SDK. What do you think?

Reply:
I like it. Also it seems to match stuff like the billing client

Reply:
is this an alternative to `public`?

Reply:
`public` is a reserved word in java and it won't let me name it like that